src/components/dialog/useFocusTrap.js

```diff
@@ -32,10 +32,7 @@ export function useFocusTrap({
     }
 
     function handleFocusTrap(event: FocusEvent) {
-      if (
-        event.target instanceof Node &&
-        dialogElement.contains(event.target)
-      ) {
+      if (event.target.nodeName && dialogElement.contains(event.target)) {
```

Comment:
This was to satisfy flow which probably won't get duck typing as a type guard I guess.